docsrc/docextract_settings.rst

```diff
@@ -45,14 +45,21 @@ Setting::
     "preset_config": "simple"
     "preset_config": "legacy"
     "preset_config": "detailed"
-
-Three preset ocr configurations are provided: ``legacy``, ``simple`` and ``detailed``. Most users will
-only need to use "simple". "legacy" is intended for users who ran Indico's original pdf_extraction
-function to extract text and train models. Use "legacy" if you are adding samples to models that were trained with
-the original pdf_extraction. "detailed" provides OCR metrics and details down to the character level- it's a lot of data.
-
-The exact settings included in "legacy", "simple" and "detailed" are shown at the bottom
-of this page.
+    "preset_config": "ondocument"
+    "preset_config": "standard"
+
+Three preset ocr configurations are provided: ``legacy``, ``simple``, ``ondocument``, ``standard``, 
```

Comment:
should we expose ondocument? It's more of an internal thing I think @Slater-Victoroff